src/mainboard/protectli/vault_jsl/devicetree.cb

```diff
@@ -54,7 +54,6 @@ chip soc/intel/jasperlake
 	# Skip the CPU replacement check
 	register "SkipCpuReplacementCheck" = "1"
 
-	register "PchHdaDspEnable" = "1"
```

Comment:
If the reason for doing this is known, would be nice to have it in the commit message.